File: subroutines/all/subroutine arrdistsd.f
      subroutine arrdistsd(a1,a2,dist,dist2)
c#    MMC routine 277 lstmod: 03/29/00
c*****dist2 = sum(ai-bi)**2
      real*8 a1,a2,dist,dist2
      real*8 x,y,z
      dimension a1(3),a2(3)
      x=a1(1)-a2(1)
      y=a1(2)-a2(2)
      z=a1(3)-a2(3)
      dist2=x*x+y*y+z*z
      dist=dsqrt(dist2)
      return
      end

File: fortran to python conversion/dependencies/subroutine arrdistsd.py
import math

def arrdistsd(a1, a2):
      """
      
      dist2 = sum(ai - bi)**2

      """
      x = a1[0] - a2[0]
      y = a1[1] - a2[1]
      z = a1[2] - a2[2]
      dist2 = x*x + y*y + z*z
      dist = math.sqrt(dist2)
      return dist